assembl/lib/model_watcher.py

```diff
@@ -67,45 +72,112 @@ def processAccountModified(self, id):
         pass
 
 
-class ModelEventWatcherPrinter(object):
+class BaseModelEventWatcher(object):
+    """A dummy :py:class:`IModelEventWatcher` for testing purposes"""
+    interface.implements(IModelEventWatcher)
```

Comment:
This is not compatible python 3. You should use `@implementer(IModelEventWatcher)` on the class.